packages/plugin-devtools/src/components/InfoComponent.ts

```diff
@@ -57,6 +57,13 @@ export class InfoComponent extends OwlComponent<{}> {
             } else {
                 return '?';
             }
+        } else if (propName === 'modifiers') {
```

Comment:
I'm not reviewing the devtools. I'm just quickly reading lines in case something really hideous catches my eye.